### DISCUSSION TITLE
Moving optex location at CTAN and TDS

### DISCUSSION THREAD

**Discussion by olsak** [olsak/OpTeX#47] [Show and tell]
OpTeX location was moved from CTAN:/macros/luatex/optex to CTAN:/macros/optex.
Contributors of OpTeX packages (if any:) should put them to CTAN:/macros/optex-contrib/package.

**Comment by vlasakm** [comment]
The location for contributed packages changed! It is now `CTAN:/macros/luatex/optex/<package name>`. This location is not often needed.

In contrast the location for contributed packages in TeX Directory Structure (the layout distributions like TeX Live use) is `TEXMF:/tex/optex/<package name>`.
